app/src/main/res/values/styles.xml

```diff
@@ -36,14 +36,38 @@
 		<item name="android:background">@drawable/keyboard_button_selector</item>
 	</style>
 
+	<style name="ListItemIcon">
+		<item name="android:layout_marginLeft">16dp</item>
+		<item name="android:layout_marginStart" tools:ignore="NewApi">16dp</item>
+		<item name="android:layout_centerVertical">true</item>
+		<item name="android:layout_alignParentLeft">true</item>
+		<item name="android:layout_alignParentStart" tools:ignore="NewApi">true</item>
+	</style>
+
 	<style name="ListItemFirstLineText">
+		<item name="android:layout_alignParentTop">true</item>
 		<item name="android:textColor">?android:textColorPrimary</item>
 		<item name="android:textSize">16sp</item>
+		<item name="android:paddingTop">20dp</item>
+		<item name="android:layout_alignParentLeft">true</item>
+	</style>
+
+	<style name="ListItemFirstLineText.WithIcon" parent="ListItemFirstLineText">
+		<item name="android:layout_marginLeft">72dp</item>
```

Comment:
I feel like maybe some of this stuff should be pulled out into constants in dimens.xml because we have a lot of repeated values.